packages/eslint-config-typescript/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@zonedigital/eslint-config-typescript",
+  "version": "0.0.0",
```

Comment:
❓ Should this be version 0.1.0?

Reply:
When we run a release with Lerna it’ll sort this out. All packages are deployed under the same version, so the version this will be, will depend on when we ship as it’ll be an iteration on what is current live.